server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -282,30 +282,7 @@ public void testExtensionsInitialize() throws Exception {
         transportService.start();
         transportService.acceptIncomingRequests();
         extensionsOrchestrator.setTransportService(transportService);
-
-        try (MockLogAppender mockLogAppender = MockLogAppender.createForLoggers(LogManager.getLogger(ExtensionsOrchestrator.class))) {
-
-            mockLogAppender.addExpectation(
-                new MockLogAppender.SeenEventExpectation(
-                    "Connect Transport Exception 1",
-                    "org.opensearch.extensions.ExtensionsOrchestrator",
-                    Level.ERROR,
-                    "ConnectTransportException[[firstExtension][127.0.0.0:9300] connect_timeout[30s]]"
-                )
-            );
-
-            mockLogAppender.addExpectation(
-                new MockLogAppender.SeenEventExpectation(
-                    "Connect Transport Exception 2",
-                    "org.opensearch.extensions.ExtensionsOrchestrator",
-                    Level.ERROR,
-                    "ConnectTransportException[[secondExtension][127.0.0.1:9301] connect_exception]; nested: ConnectException[Connection refused];"
-                )
-            );
-
-            extensionsOrchestrator.extensionsInitialize();
-            mockLogAppender.assertAllExpectationsMatched();
-        }
+        extensionsOrchestrator.extensionsInitialize();
```

Comment:
What exactly are we testing here?

Reply:
It's testing whether or not it fails from an unexpected exception

Reply:
The expected behavior for the unit test, since there isn't an extension to connect with, is to catch an exception and print a log.  Therefore, the test above will pass unless there is any other type of exception that occurs.

Reply:
I see what you're saying, will approach a different way